tests/test_format_docstring.py

```diff
@@ -1173,6 +1173,39 @@ def test_format_docstring_pre_summary_space(
             INDENTATION,
             '''\
 """This one-line docstring will have a leading space."""\
+''',
```

Comment:
We'd have to add some tests to ensure wrap lengths were set/overrode properly.

Reply:
Sure - Was going to add once the behaviour changes had been agreed

Reply:
Just need to add some tests to cover the line wrapping code you added to `configuration.py`.  Make sure to include tests for each of the config files as well as the CLI.